### PR TITLE
Log an error when the session cookie is too big

### PR DESCRIPTION
**Pull request by kumar303**
This may or may not have happened to me in production :(
I wanted to get an error so that it would go to our Sentry monitoring tool.

### CODE REVIEW

File: encrypted_cookies/__init__.py [hunk 1]
```diff
@@ -84,5 +84,14 @@ def _get_session_key(self):
         data = signing.dumps(session_cache, compress=True,
             salt='encrypted_cookies',
             serializer=EncryptingPickleSerializer)
-        log.info('encrypted session cookie is %s bytes' % len(data))
+
+        cookie_size = len(data)
+        log.info('encrypted session cookie is %s bytes' % cookie_size)
```
Comment (frankoid) [inline encrypted_cookies/__init__.py]: What would you think about making this `log.debug`? It seems quite chatty to log the size of every cookie.

Comment (kumar303) [inline encrypted_cookies/__init__.py]: sure, sounds like a good idea, especially now that there's a proper error message.

**Comment by kumar303** [comment]
hmm, I think these tests were maybe always broken? It can't find `requirements.txt`. Anyway, I can just wait until https://github.com/brightinteractive/django-encrypted-cookie-session/pull/20 lands and then rebase on that. Otherwise there will be conflicts.

**Comment by kumar303** [comment]
@frankoid I think this should be good to go now. Thanks for all the code reviews.

**Comment by frankoid** [comment]
Thanks for your contributions!

**Comment by kumar303** [comment]
Sure, glad to help. I bumped the version number for this but I guess it's not crucial (for me anyway) to release it to PyPI. If you do release it just ping me so I can upgrade my dependencies.

**Pull request merged**